src/Responsive/index.jsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
-import Responsive from 'react-responsive';
+export {
```

Comment:
Rather than having these exported from `Responsive/index.jsx`, I might lean towards having `src/index.js` be responsible for the exports from `react-responsive`.

That would leave just the `breakpoints` in this file, at which point these could probably be moved to its own standalone file `src/utils/breakpoints.js`.

The Paragon components that make use of react-responsive could then still get their responsive imports directly from `react-responsive` itself, which is likely a bit more clear and understandable.

Curious to get to get your take, of course! :)

Reply:
I think that structure makes a lot more sense - thanks!